### PR TITLE
Bump up Prometheus stack version

### PR DESCRIPTION
**Pull request by cityofships**
Related: https://github.com/stackhpc/stackhpc-kayobe-config/pull/67

### CODE REVIEW

**Review by markgoddard — APPROVED** [review]

File: etc/kayobe/kolla/globals.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ nova_tag: wallaby-20220328T103755
 octavia_tag: wallaby-20220524T085900
 ovn_tag: wallaby-20220223T143249
 openvswitch_tag: wallaby-20220223T143249
+prometheus_tag: wallaby-20220525T174700
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml:18]: Did you rebuild all prometheus images? Should we remove the ones below?

**Review by markgoddard — CHANGES_REQUESTED** [review]
Waiting for answer

File: etc/kayobe/kolla/globals.yml (hunk 1 above)
Comment (cityofships) [reply etc/kayobe/kolla/globals.yml:18]: The regexp feature is broken, thus all get rebuilt always. Fix in https://github.com/stackhpc/stackhpc-kayobe-config/pull/68
Does ~30 minutes build time of all images match your experience? I saw other jobs that had run for ~4h 30m

Comment (markgoddard) [reply etc/kayobe/kolla/globals.yml:18]: 30 minutes would be pretty quick, even if using the cache.

Comment (markgoddard) [reply etc/kayobe/kolla/globals.yml:18]: Presumably you at least want to upgrade alertmanager, given you changed the version

Comment (cityofships) [reply etc/kayobe/kolla/globals.yml:18]: By default: ```prometheus_alertmanager_tag: "{{ prometheus_tag }}"```
The ones below are pretty custom - I'm not confident they'll work correctly in the newer build, thus leaving them.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**